inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -119,6 +145,22 @@ void FrontEnd::parseStaticShapeNMS(const Model& model, const ie::CNNLayerPtr& la
 
     auto stage = model->addNewStage<StaticShapeNMS>(layer->name, StageType::StaticShapeNMS, layer, usedInputs, DataVector{outIndices, outShape});
     stage->attrs().set<bool>("center_point_box", centerPointBox);
+
+    const auto inputDims0 = inputs[0]->desc().dims();
+    const auto perm = DimsOrder::fromNumDims(inputDims0.size()).toPermutation();
+    const auto spatDim = inputDims0[perm[1]];
+
+    const int ddrBufferSize0 = 2 * sizeof(int16_t) * 4 * spatDim;
+    const int ddrBufferSize1 = 2 * sizeof(int16_t) * spatDim;
+    const int ddrBufferSize2 = 2 * sizeof(int32_t) * spatDim;
+    const int ddrBufferSize = ddrBufferSize0 + ddrBufferSize1 + ddrBufferSize2 + 2 * vpu::DATA_ALIGNMENT;
+
+    const auto& env = CompileEnv::get();
+    const auto numSlices = env.resources.numSHAVEs;
+
+    const int cmxBufferSize = 4 * sizeof(int32_t) * 256;
```

Comment:
There is 2 different cmxBufferSize in this method. What is the difference?

Reply:
`CMX_BUFFER_SIZE` - size of available CMX for one SHAVE to use. cmxBufferSize - size of allocated buffer on CMX by SHAVE. Renamed variables for clarification.